Makefile

```diff
@@ -87,6 +93,9 @@ go-lint: ## Lint Go code
 gosec: ## Scan Go code for security flaws
 	docker compose run --rm gosec
 
+go-test: ## Run go unit tests
+	go test ./... -race -covermode=atomic -coverprofile=coverage.out
```

Comment:
Looks like we already have this as `make test` on line 6?

Reply:
Ah, muscle memory from MRLPA 😅 I'll remove this.